### PR TITLE
Add agility stat

### PR DESCRIPTION
**Pull request by OcelotWalrus**
This PR makes so that every weapon and armor items have an agility stat that is added together to make the player agility stat.

The enemy also have a random agility stat so both you and your enemy have some chance to dodge an attack, which will result in taking no damage for either you or your enemy.

Everything works but I'll make enemy lists today and other stuff we've discussed in #15 yesterday.

As soon as this PR is merged, I'll work on enhancing enemies.

### CODE REVIEW

File: battle.py [hunk 1]
```diff
@@ -33,20 +35,28 @@ def fight(player, item):
             # print HP stats and possible actions for the player
 
             print(str(COLOR_RED) + "Enemy: " + str(enemy_health) + str(COLOR_RESET_ALL) + "/" + str(COLOR_GREEN) + str(enemy_max) + str(COLOR_RESET_ALL) + "; " + str(COLOR_BLUE) + "You: " + str(player["health"]) + str(COLOR_RESET_ALL) + "/" + str(COLOR_GREEN) + str(player["max health"]) + str(COLOR_RESET_ALL))
-            action = input("Attack, Defend, Use Item? ")
+            action = input("(A)ttack, (D)efend, (U)se Item? ")
```
Comment (lumbar527) [inline battle.py:38]:
```suggestion
            action = input("[A]ttack, [D]efend, [U]se Item? ")
```

**Review by lumbar527 — APPROVED** [review]

File: battle.py (hunk 1 above)
Comment (OcelotWalrus) [reply battle.py:38]: Sure!

**Comment by OcelotWalrus** [comment]
Please could you merge it so I can start working on the enemies enhancement?

**Comment by lumbar527** [comment]
Merging...

**Comment by OcelotWalrus** [comment]
Thanks!

**Comment by lumbar527** [comment]
You're welcome!

**Pull request merged**